### ISSUE TITLE
Scoped App Explorer not working

### ISSUE THREAD

**Issue by Eiluviann** [arnoudkooi/sn-scriptsync#73]
Looks like scoped app explorer is not working and the last commit was 3 years ago with a comment "Work in Progress". Was this feature abandoned? If so why is the tab still visible in the current versions?

**Comment by arnoudkooi** [comment]
It should work, it is triggered from Studio.
Have you cheked the video?
https://youtu.be/ZDDminMjGTA?t=423

Let me know if it works, and if this is what you meant.

**Comment by MickeyHuntr** [comment]
When I follow the steps in the video nothing really happens. The list shows categories but nothing is under them.
![image](https://user-images.githubusercontent.com/40126502/184319120-0c147fe6-3fb3-47ee-b236-5c15dc0fafc6.png)


When I try to use "Open Scope File Selection" an error appears
![image](https://user-images.githubusercontent.com/40126502/184318989-bf4a0025-b5ea-4651-ac2b-38a79f2b29ea.png)

**Comment by arnoudkooi** [comment]
Did you try a reinstall? Do you have more context on browser setup os etc.?

**Comment by MickeyHuntr** [comment]
Hi, sorry for late response.
I'm running MacOS Monterey 12.4, VSCode 1.70.1 and sn-scriptsync v2.3.3

**Comment by arnoudkooi** [comment]
Please add a screenshot of your studio environment where you click the link button, and details of browser setup as well

**Comment by MickeyHuntr** [comment]
Unfortunately I cannot share a screenshot since it contains a lot of classified data but I tried running scoped files again today and I see that the list got **somewhat** populated.

Forms & UI only shows UI macros and under server development there are only script includes. The rest of the categories/subcategories are empty. By looking at the studio I see that there are a lot of files that should be on the list but are not (of course I do not count in records which aren't scripts). 

When I tried this functionality on my personal instance nothing would appear on the list since all I had was pages, angular providers and widgets. Pages are records without scripts, angular providers do not show up in studio but widgets should have been synced.
